packages/csv-parser-price/src/cli.js

```diff
@@ -85,14 +87,12 @@ Convert commercetools price CSV data to JSON.`,
   })
 
   .option('logLevel', {
-    default: 'info',
     describe: 'Logging level: error, warn, info or verbose.',
   })
-  .coerce('logLevel', (arg) => {
-    npmlog.level = arg
-  })
   .argv
 
+npmlog.level = args.logLevel || minLogLevel
```

Comment:
We already had a working feature of this before, please refer to this [line](https://github.com/commercetools/csv-parser-price/blob/master/src/cli.js#L54) to see how to improve this